src/main/java/com/cgi/ipsen5/Controller/UserController.java

```diff
@@ -77,7 +80,14 @@ public ApiResponse<List<UserFavoriteColleaguesDTO>> getFavoriteColleaguesFromAct
     public void addFavoriteColleague(@RequestBody UserFavoriteColleaguesDTO favoriteColleaguesDTO) {
         Authentication authentication = SecurityContextHolder.getContext().getAuthentication();
         this.favoriteColleagueService.addFavoriteColleague(UUID.fromString(authentication.getPrincipal().toString()),
-                favoriteColleaguesDTO.getIdOfFavorite());
+                favoriteColleaguesDTO.getId());
+    }
+
+    @DeleteMapping(path = "/favorite-colleagues")
+    public void removeColleagueFromFavorites(@RequestBody UserFavoriteColleaguesDTO favoriteColleaguesDTO){
+        Authentication authentication = SecurityContextHolder.getContext().getAuthentication();
+        this.favoriteColleagueService.removeColleagueFromFavorites(UUID.fromString(authentication.getPrincipal().toString()),
+                favoriteColleaguesDTO.getId());
```

Comment:
is het niet beter om status code terug te sturen om te kijken of die req gelukt is